packages/form-js/CHANGELOG.md

```diff
@@ -6,6 +6,20 @@ All notable changes to [form-js](https://github.com/bpmn-io/form-js) are documen
 
 ___Note:__ Yet to be released changes appear here._
 
+## 0.10.0
```

Comment:
We didn't squash the alpha notes once we released the stable in the past, so not sure whether we should do it this time. Let me know what you think. @Skaiir @vsgoulart